tests/src/testenhancedassumedstrains.cpp

```diff
@@ -31,16 +28,23 @@ int main(int argc, char** argv) {
   constexpr auto randomlyDistorted      = CornerDistortionFlag::randomlyDistorted;
   constexpr auto unDistorted            = CornerDistortionFlag::unDistorted;
 
-  t.subTest(testFEElement<EASElement>(firstOrderLagrangePrePower2Basis, "EAS", randomlyDistorted,
-                                      Dune::ReferenceElements<double, 2>::cube(), checkJacobianFunctor));
+  auto linearElasticEASFunc = [](const Ikarus::YoungsModulusAndPoissonsRatio& mat) {
```

Comment:
I think this should be called only `linearElasticFunc` because there is nothing specific to `EAS` in it